python/parser.js

```diff
@@ -136,7 +135,13 @@
             string = `${JSON.stringify(region)}`
             break;
         case "object":
-            string = parseObject(region.type ? region : (region.shadow ? region : {value: region, type:'Region'}))
+            if (region.region) {
+                string = python`${region.region}, padding=${region.padding}`
+            } else {
+                string = parseObject(
+                    region.type ? region : (region.shadow ? region : {value: region, type:'Region'})
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9697038)
```suggestion
                    region.type ? region : (region.shadow ? region : {value: region, type:"Region"})
```